runbotics/runbotics-orchestrator-ui/src/src-app/types/format.ts

```diff
@@ -0,0 +1 @@
+export const VARIABLE_NAME_PATTERN = /(^\$.+)|(^#.+)/;
```

Comment:
Regex is invalid
<img width="146" alt="image" src="https://github.com/runbotics/runbotics/assets/44839807/5a45074b-bd5e-4481-a154-52fc7b86e016">